src/common/__tests__/data/courses.ts

```diff
@@ -117,3 +117,22 @@ export const updateCourseExample: UpdateCourseDTO = {
   isSEAS: IS_SEAS.Y,
   private: true,
 };
+
+/**
+ * A collection of catalog prefix data, as returned by the database query in
+ * [[CourseService#getCatalogPrefixList]]
+ */
+export const rawCatalogPrefixList = [
+  { prefix: 'AC' },
+  { prefix: 'AM' },
+  { prefix: 'AP' },
+  { prefix: 'BE' },
+  { prefix: 'CS' },
+  { prefix: 'EE' },
+  { prefix: 'ESE' },
+  { prefix: 'General' },
+  { prefix: 'Mat & ME' },
+  { prefix: 'MDE' },
+  { prefix: 'MSMBA' },
+  { prefix: 'SEM' },
+];
```

Comment:
A couple of these (e.g. `'EE'`, `'Mat & ME'`) aren't actually course prefixes, but rather Areas.  It's a minor point, but as always we want our test data to mirror reality as much as possible.

Reply:
Ah good point! I should have checked the actual app. I updated the test data in 14cd59c so that it matches the current app's catalog prefixes in the "4 Year Plan" tab.